controllers/topics.controller.js

```diff
@@ -1,12 +1,16 @@
 const { fetchTopic } = require("../models/topics.model");
 
-exports.getTopics = (req, res, next) => {
-    const { topics } = req.params;
-    fetchTopic(topics)
+const getTopics = (req, res, next) => {
+  const { topics } = req.params;
+  fetchTopic(topics)
     .then((topic) => {
-        res.status(200).send({ topics: topic })
+      res.status(200).send({ topics: topic });
```

Comment:
Are you sending 'topics' or a single 'topic' back here? Please be consistent with naming.